repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/module-setup.sh

```diff
@@ -73,6 +76,13 @@ install() {
 
     # install this one to ensure we are able to sync write
     inst_binary sync
+    # install in-band debugging utilities
+    inst_binary base64
+    inst_binary tar
+    inst_binary xz
+    inst_binary sha256sum
```

Comment:
I don't see `sha256sum` being used anywhere in the code, is it a dependency of a dependency?

Reply:
Good point, removing.